usagov/vote-gov-drupal
# VOTE-587 state selector paragraph refinements

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-587

## Description

State selector (Registration tool) paragraph refinements

## Deployment and testing

### Post-deploy

1. cd into votegov theme and run `npm run build`
2. run `lando retune`

### QA/Test

1. Edit the homepage http://vote-gov.lndo.site/node/63/edit and add a placeholder value in the registration tool component.
2. Visit the homepage http://vote-gov.lndo.site/ and check if the placeholder field is displaying correctly in the text field of the component

## Checklist for the Developer




## Checklist for the Peer Reviewers
